### ISSUE TITLE
Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send

### ISSUE THREAD

**Issue by Ashutosh-Spark** [umair13adil/flutter_logs#48]
We are getting this error while exporting logs for android devices.
This is happening if the device is kept still for some time . We are able to reproduce this everytime.

**Comment by Abhinav-Satija** [comment]
Yes, We are getting this error while exporting logs for android devices.

**Comment by word-x** [comment]
> Yes, We are getting the same error while exporting logs for android devices.

Hi, have you got it fixed?
